libjulia/backends/evm/EVMAssembly.cpp

```diff
@@ -77,6 +77,13 @@ EVMAssembly::LabelID EVMAssembly::newLabelId()
 	return m_nextLabelId++;
 }
 
+AbstractAssembly::LabelID EVMAssembly::namedLabel(string const& _name)
+{
+	if (!m_namedLabels.count(_name))
```

Comment:
Hm, should this have an assert for `!name.empty()`?

Reply:
Why should we handle that case specially?

Reply:
I am a bit confused? Only meant that we should reject an empty name. Applies to this and the libevmasm.

Reply:
Sure, but why should we reject an empty name?

Reply:
What reason is there to allow it? If for nothing else, printing it would be an issue (how to properly refer to an empty tag?).

I think it can also help it finding bugs where unintentionally an empty name was pushed.